integration/tests/tracer/test_tracer_debug_methods.py

```diff
@@ -254,4 +255,164 @@ def test_debug_trace_block_by_non_existent_hash(self):
             method="debug_traceBlockByHash", params=['0xd97ff4869d52c4add6f5bcb1ba96020dd7877244b4cbf49044f49f002015ea85'])
         assert "error" in response, "No errors in response"
         assert response["error"]["code"] == -32603, "Invalid error code"
-        assert response["error"]["message"] == "eth_getBlockByHash returns None for '\"0xd97ff4869d52c4add6f5bcb1ba96020dd7877244b4cbf49044f49f002015ea85\"' block"
\ No newline at end of file
+        assert response["error"]["message"] == "eth_getBlockByHash returns None for '\"0xd97ff4869d52c4add6f5bcb1ba96020dd7877244b4cbf49044f49f002015ea85\"' block"
+
+    def test_getRawHeader_by_block_number(self):
+        receipt = self.send_neon(
+            self.sender_account, self.recipient_account, 0.1)
+        assert receipt["status"] == 1
+        wait_condition(lambda: self.tracer_api.send_rpc(method="debug_getRawHeader",
+                                                        params=[hex(receipt["blockNumber"])])["result"] is not None,
+                       timeout_sec=120)
+        
+        response = self.tracer_api.send_rpc(
+            method="debug_getRawHeader", params=[hex(receipt["blockNumber"])])
+        assert "error" not in response, "Error in response"
+        assert response["result"] is not None and response["result"] != ""
+        assert isinstance(response["result"], str)
+
+    def test_getRawHeader_by_invalid_block_number(self):
+        response = self.tracer_api.send_rpc(
+            method="debug_getRawHeader", params=["0f98e"])
+        assert "error" in response, "No errors in response"
+        assert response["error"]["code"] == -32602, "Invalid error code"
+        assert response["error"]["message"] == "Invalid params"
+    
+    def test_getRawHeader_by_block_hash(self):
+        receipt = self.send_neon(
+            self.sender_account, self.recipient_account, 0.1)
+        assert receipt["status"] == 1
+        wait_condition(lambda: self.tracer_api.send_rpc(method="debug_getRawHeader",
+                                                        params=[receipt["blockHash"].hex()])["result"] is not None,
+                       timeout_sec=120)
+        
+        response = self.tracer_api.send_rpc(
+            method="debug_getRawHeader", params=[receipt["blockHash"].hex()])
+        assert "error" not in response, "Error in response"
+        assert response["result"] is not None and response["result"] != ""
```

Comment:
check body too. Because not blank response is not verify that response work good

Reply:
done